### DISCUSSION TITLE
Feature Request: Option to Stack Time Series Charts

### DISCUSSION THREAD

**Discussion by kk6mrp** [grafana/grafana#55006] [Dashboards, Panels & Visualizations]
It would be very useful to be able to stack the trends on the Time Series Chart in order to better see correlation between changing values. Like in the image below, each data point is stacked one above the other, each showing their respective scale.

This makes it easy to see how small increases of small values have an effect on large values. For example, a boolean input switching on has a large effect on an analog value.

![image](https://user-images.githubusercontent.com/9358855/189466097-80c8faa0-f3ab-4359-9bd5-f8a45d69f072.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
